source/draw/RasterMonochrome.ooc

```diff
@@ -43,6 +43,40 @@ RasterMonochrome: class extends RasterPacked {
 		if (monochrome != image)
 			monochrome referenceCount decrease()
 	}
+	// Precondition: Do not draw out of bound
+	_blitWhiteRaw: static func (target, source: Byte*, startSrcX, startSrcY, startDstX, startDstY, width, height, targetStride, sourceStride: Int) {
```

Comment:
That would give highly unreadable code and defeat the purpose of the convention since this is a helper function that must be close to the parent function.